src/main/groovy/life/qbic/datamodel/dtos/projectmanagement/ProjectSpace.groovy

```diff
@@ -0,0 +1,47 @@
+package life.qbic.datamodel.dtos.projectmanagement
+
+/**
+ * Represents a QBiC project space.
+ *
+ * A space is a logical grouping of projects that have the same context.
+ *
+ * A project space has a name that follows this convention:
+ *
+ *  1. No white space
+ *  2. Capital letters only
+ *  3. Inner white space is replaced by `_`
+ *  4. Trailing and leading white space are trimmed
+ *
+ * @since 2.3.0
+ */
+final class ProjectSpace {
+
+    /**
+     * The project space name
+     */
+    final String name
```

Comment:
Is there any rule on the name. What does it contain? E.g Customer name + project name or so?

Reply:
Rule see the format space name and class description. There is an outdated "rule" the project managers use, but as we discussed in the stakeholder interview, this was merely a hack, because they were not able to filter otherwise. we have the properties available in the offer (the selected customer affiliation).